cs/src/core/Clone.cs

```diff
@@ -37,30 +37,58 @@ public static T From<SourceT>(SourceT source)
     public class Cloner<SourceT>
     {
         readonly Func<object, object>[] clone;
-        
+
+        /// <summary>
+        /// Create a cloner that makes clones of the same type SourceT as source objects.
```

Comment:
Minor: Can we use the `typeparamref` for `SourceT`, please?
This needs to be done in other places as well.